src/get-started/java-binding.md

```diff
@@ -240,6 +240,33 @@ at `/cryptocurrency/balance/:walletId`.
 See [documentation][vertx.io] on the possibilities of `Vert.x` used as a web
 framework.
 
+### Supported Services
+
+Currently Java Binding supports the following services:
+
+- **Configuration Update Service.**
+  Although every node has its own configuration file, some settings should be
+  changed for all nodes simultaneously. This service allows updating
+  configuration through the blockchain itself.
```

Comment:
True, but the main purpose of configuration service is to update blockchain configuration on-the-flight, without stopping nodes. Any other operation is possible using regular configuration files.

Reply:
How about rephrasing the second sentence as follows:
> The present service allows updating global configuration parameters of the network without stopping the nodes. The changes are agreed upon through the consensus mechanism.

??